hello-world-python/README.md

```diff
@@ -0,0 +1,110 @@
+
+Python is not enabled by default. 
+In order to enable support add the following line to your enclave build.gradle.
+
+      conclave {
+      productID = 1
+      ...
+      supportLanguages = "python"
+      }
+
+Adding this line pulls all the required python dependencies.
+
+Let's define our python function which we want to execute inside the enclave in the ReverseEnclave.class
+
+      private static final String pythonCode = "def reverse(input):\n"
+      + " return input[::-1]";
+
+Let us also define the Bindings and the Context 
+
+      private final Context context;
+      private final Value bindings;
+
+Let us now define the bindings in the constructor
+
+      public ReverseEnclave() {
+      context = Context.create("python");
+      bindings = context.getBindings("python");
+      context.eval("python", pythonCode);
+      }
+
+This sets up the bindings to Java which allow us to access the functions and variables that are defined within the 
+Python code.
+
+Below code snippet shows you how we can execute the function binded by use before.
+
+      private String reverse(String input) {
+        Value result = bindings.getMember("reverse").execute(input);
+        return result.asString();
+    }
+
+Now we are done with adding the configurations and now let's see hwo to run this sample in mock mode.
+
+# Steps to run the Hello World python sample in mock mode
+
+For this sample, we will run our sample in mock mode.
+There is development work going on to make it work in the simulation and other modes.
+Before running this sample in mock mode make sure to perform below steps.
+
+1. Download [graalvm-ce-java11](https://github.com/graalvm/graalvm-ce-builds/releases). 
+2. Set the environment variable JAVA_HOME to point to the GraalVM that was previously downloaded. 
+   For instance, export JAVA_HOME=/usr/lib/jvm/graalvm-ce-java11-21.1.0).
+3. Update the environment variable PATH by running export PATH=JAVA_HOME/bin:$PATH.
+4. For Python only - Install the Python component by running the command gu install python.
+
+Once you have set GraalVM as your JAVA_HOME, we will build and start the host and the client
+
+Start the host by executing below command, which will build the enclave and host:
+
+```bash
+./gradlew host:bootJar
+java -jar host/build/libs/host-mock.jar
+```
+
+It should print out some info about the started enclave. Then you can use the client to send it strings to reverse:
+
+```bash
+./gradlew client:shadowJar
+java -jar client/build/libs/client.jar "S:0000000000000000000000000000000000000000000000000000000000000000 PROD:1 SEC:INSECURE" "reverse me"
+```
+
+## Next step
+For more information on how to run a python code in Conclave take a look at the [docs](https://docs.conclave.net/javascript-python.html).
+Take a look at 
+
+This is a simple app using the Conclave API. It is licensed under the Apache 2 license, and therefore you may
```

Comment:
Is this really needed here?

Reply:
removed, it's good to keep it simple and clean.